documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -2,166 +2,367 @@
 
+   These parameters are specified by inclusive edge limits.
 
-where ``pbin_1``, ``pbin_2``,... have the same form as for cut_sqw described above, and there are as many binning arguments as the dimensionality of the object. This is essentially the same syntax as ``cut_sqw``, but with an sqw object rather than an SQW file as the first input. The same set of optional arguments, namely ``'-nopix'`` and ``filename`` also applies.
+.. warning::
+
+   Selected bins will be those whose bin centres lie within the range ``lo - hi``, this means that the actual returned
+   ``img_range`` may not match ``[lo hi]``. For example, a bin from ``0 - 1`` (centre ``0.5``) will be included by the
+   following ``section`` despite the bin not being entirely contained within the range. The resulting image range will
+   be ``[0 1]``.
+
+   .. code-block:: matlab
+
+      section(w, [0.4 1])
+
+In order to extract bins whose centres lie in the range ``[-5 5]`` from a 4-D ``sqw`` object:
+
+.. code-block:: matlab
+
+   w2 = section(w1, [-5 5], [], [], [])
 
 
 head_horace
 ===========
 
-::
+.. code-block:: matlab
 
-   info=head_horace(filename);
+   info = head_horace(filename);
 
-   info=head_horace(filename,'-full')
+   info = head_horace(filename,'-full')
 
 
-This is a function to give the header information in an SQW file or file to which an sqw object or dnd object has been saved, and whose full filename is given by the argument ``filename``. If the option ``'-full'`` is used then a fuller set of header information, rather than just the principal header, is returned. The purpose of this function is to read the contents regardless of your knowledge of whether or not the file contains an sqw object or a dnd object.
+This is a function to give the header information in an SQW file or file to which an sqw object or dnd object has been
+saved, and whose full filename is given by the argument ``filename``. If the option ``'-full'`` is used then a fuller
+set of header information, rather than just the principal header, is returned. The purpose of this function is to read
+the contents regardless of your knowledge of whether or not the file contains an sqw object or a dnd object.
 
 
 head_sqw
 ========
 
-::
+.. code-block:: matlab
 
-   info=head_sqw(filename);
+   info = head_sqw(filename);
 
-   info=head_sqw(filename,'-full')
+   info = head_sqw(filename,'-full')
 
 
-This is a function to give the header information in an SQW file or file to which an sqw object has been saved, whose full filename is given by the argument ``filename``. If the option ``'-full'`` is used then a fuller set of header information, rather than just the principal header, is returned.
+This is a function to give the header information in an SQW file or file to which an sqw object has been saved, whose
+full filename is given by the argument ``filename``. If the option ``'-full'`` is used then a fuller set of header
+information, rather than just the principal header, is returned.
```

Comment:
an example might be a good idea

Reply:
I have no idea what this does. This is old stuff